po/ca.po

```diff
@@ -980,6 +980,128 @@ msgstr ""
 "Proveu de combinar-la amb tecles com la de Control, Majúscula o "
 "Alternativa.\n"
 
+#: guake/guake_app.py:491
+msgid ""
+"Guake is now running,\n"
+"press <b>{!s}</b> to use it."
+msgstr ""
+
+#: guake/main.py:153
+msgid "Return the selected tab label."
+msgstr ""
```

Comment:
I think we can avoid removing existing translation (for the french one, it pretty much kept all ones)

Reply:
I don't think that it deleted any translations. Did I miss something? The git-diff shows me only additions to the .po files but no deletions/changes.

Reply:
Ah ok. I know why we "lost" some of the translation 🤦‍♂️ the new version of the glade files does not separate combobox items by line-break but has individual xml elemets for all items (for example).

```
Never\n
With process running\n
Always
```
Versus
```
<item translatable="yes">Never</item>
<item translatable="yes">With process running</item>
<item translatable="yes">Always</item>
``` 

I have to spend some time on this! 

Info for myself: [--use-first](https://github.com/Guake/guake/blob/6a3cc349256871ff104733378f27a801182624f0/Makefile#L138) should maybe also changed.

Reply:
Hum, after the merge the translation hasn't been lost :)